redis/Module.md

```diff
@@ -0,0 +1,102 @@
+## Overview
+
+Ballerina connector for Redis is connecting Redis via Ballerina language easily. This module provides the functionality required to access and manipulate key-value data stored in a Redis database this module provides API functions for manipulating Redis String, Hash, List, Set and Sorted Set data types.
+Currently the provided functionality is limited to usage Redis as a database/cache. Broker functionality of Redis is not yet exposed through this module.
+
+## Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+* Make sure redis cache is running in local environment or use redis cache running on cloud such as [Redis Enterprise Cloud](https://redislabs.com/redis-enterprise-cloud/overview/)
```

Comment:
Suggestion:
Make sure Redis cache is running in the local environment or use Redis cache running on the cloud (for example, [Redis Enterprise Cloud](https://redislabs.com/redis-enterprise-cloud/overview/)).